rising/utils/checktype.py

```diff
@@ -9,4 +11,9 @@ def check_scalar(x):
     Returns:
         True if input is scalar
     """
-    return isinstance(x, (int, float))
+    if isinstance(x, (int, float)):
+        return True
+    elif torch.is_tensor(x):
```

Comment:
don't use this :) instead use `is instance(x, torch.Tensor)`, which is internally called by `torch.is_tensor`. `torch.is_tensor` is a legacy API from the time when tensors were no native python class

Reply:
👍 Learned something new :)